pkg/session_recording/recordings.jsx

```diff
@@ -193,13 +217,77 @@
             });
         }
 
+        sendToTerm(value) {
+            this.refs.terminal.state.terminal.send(value);
+        }
+
+        playPauseToggle() {
+            if (this.state.playback === 'pause') {
+                this.setState({playback: 'play'});
+            } else {
+                this.setState({playback: 'pause'});
+            }
```

Comment:
If we switch to the boolean `this.state.paused`, this can become just `this.setState({paused: !this.state.paused})`.